config/game_options.txt

```diff
@@ -502,3 +502,11 @@ MAXFINE 2000
 
 ## Whether native FoV is enabled for all people.
 #NATIVE_FOV
+
+## SKYRAT ADDITION START
```

Comment:
there is a skyrat config (config/skyrat/skyrat_config.txt)

Reply:
based